### DISCUSSION TITLE
MacOS compatibility information?

### DISCUSSION THREAD

**Discussion by snarke** [Hammerspoon/hammerspoon#3199] [Q&A]
I'm trying to get my Griffin PowerMate USB knob working on my computer after upgrading from MacOS 10.11 to (officially unsupported on a MacPro3,1) 10.14. I found a third-party driver that seems to be a thing that uses Hammerspoon. So now I'm here. Downloaded the newest version, but it's marked by the OS as "not runnable." Seems a safe bet the current version wants MacOS 11 or 12 or some otherwise shiny new version of the OS. But there doesn't seem to be any information in the documentation about what the actual requirements of any currrent or previous Hammerspoon ARE. I can try to guess based on the year a particular version was released, if I can figure out how to download an older version. Something more specific or concrete would be very helpful.

**Comment by latenitefilms** [comment]
[v0.9.91](https://github.com/Hammerspoon/hammerspoon/releases/tag/0.9.91) should still work on Mojave.

**Comment by dasmurphy** [comment]
I got 2 things to say:

[Here](https://github.com/Hammerspoon/hammerspoon/issues/3023) is a 0.9.93 version linked which works in Mojave.

And [here](https://github.com/Hammerspoon/hammerspoon/issues/3020#issuecomment-992526342) is explained which operating system is dropped...

> we typically only support the current release and two previous versions

Have fun :)
